tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "ES2019",
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "removeComments": false,
+    "strict": true,
+    "alwaysStrict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "allowJs": true,
+    "outDir": "./",
+    "typeRoots": [
+      "node_modules/@types"
+    ]
+  },
+
+  "include": ["./src/*.js", "./src/*.ts"],
+  "exclude": []
```

Comment:
I don't think this is required.

Reply:
I recall getting an error when pulling this out of a tsconfig for one of our core packages, so interested to know if it is truly necessary

Reply:
Without it, running tsc results in: 

"error TS18003: No inputs were found in config file '/Users/kovarik/workspace/cumulus-message-adapter-js/tsconfig.json'. Specified 'include' paths were '["src"]' and 'exclude' paths were '["./"]'."

The reason why is: 

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html

"Files included using "include" can be filtered using the "exclude" property. However, files included explicitly using the "files" property are always included regardless of "exclude". The "exclude" property defaults to excluding the node_modules, bower_components, jspm_packages and <outDir> directories when not specified."

----

This allows the /src dir to be a subdir of outDir.   

I'd pulled that pattern in from some of the Core modules, but I don't believe there's a reason I can't target a /dist directory with the tests and packaging if we so desire.

Reply:
also see comment at L17 97ca78b

Reply:
After moving `outDir` to `dist`, though, the `exclude` should no longer be needed.

Reply:
Right, it's been removed due to that.